### PR TITLE
Goonetics fixes

### PR DESCRIPTION
**Pull request by Time-Green**
:cl:
fix: fixes monkey and species mutations not working
fix: fixes not getting gibbed
/:cl:
apparently 
```
if(10 to 5)
```
doesnt ever do anything
PRB: no update

### CODE REVIEW

**Comment by Vidiveni** [comment]
Injectors last forever btw. Don't know if that's been reported or not, but it does not feel intended. Makes activators pointless.

**Comment by Vidiveni** [comment]
We should see how it plays out first I guess. 

I just had an awful round on Terry today where someone kept griefing me with telekinesis, and I had no way to find out who since people had telekinesis all over the place. People using telekinesis on chem dispensers to create space lube smoke apparently isn't logged also. Last fingerprint gave my own ckey instead (last to physically touch the dispenser). 

Basically it felt too easy to distribute powers, since you only needed to click yourself with a mutator without going through the old rad shower. But we'll see how it plays out. It was a bit of a test round I suppose.

**Comment by Time-Green** [comment]
> We should see how it plays out first I guess.
> 
> I just had an awful round on Terry today where someone kept griefing me with telekinesis, and I had no way to find out who since people had telekinesis all over the place. People using telekinesis on chem dispensers to create space lube smoke apparently isn't logged also. Last fingerprint gave my own ckey instead (last to physically touch the dispenser).
> 
> Basically it felt too easy to distribute powers, since you only needed to click yourself with a mutator without going through the old rad shower. But we'll see how it plays out. It was a bit of a test round I suppose.

I think I was in that round, that is if the clown had about every mutation and there were multiple of him. 
Telekinesis will be harder to get once I lock it behind some other new mutations
You raise a fair point though

**Comment by Time-Green** [comment]
Could we merge this? Genetics is kinda easy now since stacking powers got readded by accident
